pkgs/development/python-modules/async-tkinker-loop/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, pkgs
+, python3
+, fetchPypi
+}:
+python3.pkgs.buildPythonPackage rec {
+  pname = "async-tkinker-loop";
+  version = "0.8.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "async_tkinter_loop";
+    hash = "sha256-+9AvnYIZMWCbpCEKdbIadyU8zgyUlW/fRYYyDOxAzeg=";
+  };
+
+  buildInputs = with python3.pkgs; [
+    poetry-core
+  ];
+
+  patches = [
+    ./asyncio-built-in.patch
+  ];
+
+  meta = with lib; {
```

Comment:
Would you please add the changelog to the meta?

Reply:
Absolutely. Somehow only just learned of its existence today :)